controllers/nc-news-controller.js

```diff
@@ -0,0 +1,13 @@
+const endpointsJson = require("../endpoints.json")
+const selectTopics = require("../models/nc-news-model")
+
+function getApi(req, res){
+    res.status(200).send({ endpoints: endpointsJson });
+}
+function getTopics(req, res) {
+    selectTopics().then((topics) => {
+      res.status(200).send({ topics });
+    });
+  }
+
+module.exports = {getApi, getTopics}
```

Comment:
Good work in the controller, however it is better to separate your controllers into what they actually do, for example, here, you have called it "nc-news-controller.js". But you might want to separate them into "topics.controllers.js" and an other one to deal with something else like articles